spec/features/availability_spec.rb

```diff
@@ -86,7 +86,7 @@
   end
 
   describe 'Archival Material:' do
-    it 'has a \'Request Material\' link so it can be requested through Aeon', js: true do
+    it 'has a "Request Material" link so it can be requested through Aeon', js: true do
```

Comment:
oh yeah I changed this too, just because I saw it and it seemed like an unneeded escaping